app/cdap/components/WrangleHome/Components/WrangleCard/iconStore/CloudSQLPostGreSQL.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+export const CloudSQLPostGreSQL = (
```

Comment:
do we need to import all these svgs?

Reply:
Done

Reply:
No I mean, why are we importing all these svgs?